packages/lime-proto-babeld/files/usr/lib/lua/lime/proto/babeld.lua

```diff
@@ -77,9 +82,9 @@ function babeld.configure(args)
 	uci:set("babeld", "localdeny", "action", "deny")
 
 	-- Avoid redistributing enything else
-	uci:set("babeld", "dany", "filter")
-	uci:set("babeld", "dany", "type", "redistribute")
-	uci:set("babeld", "dany", "action", "deny")
+	uci:set("babeld", "deny", "filter")
```

Comment:
`dany` here was a contraption of "deny any", as this is the `uci` section name it should not change the sematic, but I hope we don't get some naming collision in the future :p

Reply:
:facepalm: ok, maybe rename it to deny_any?

Reply:
yeah that seems better, could you do that before merging this?